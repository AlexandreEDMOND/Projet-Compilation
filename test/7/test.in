int main()
{
  float a;
  float b = 1856.75982;
  float c = -86.34;

  a = 45.26;
  b = -84.7;
  c = 0.0001;

  print(a);
  print(b);
  print(c);
  
  return 0;
}