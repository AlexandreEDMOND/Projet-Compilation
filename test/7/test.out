45.25999832
-84.69999695
0.00010000
